changelog/config.yaml

```diff
@@ -15,5 +15,5 @@ scopes:
   programgen: [dotnet, go, java, nodejs, python, yaml]
   protobuf: []
   sdk: [dotnet, go, java, nodejs, python, yaml]
-  sdkgen: [dotnet, go, java, nodejs, python, yaml]
-  yaml: []
\ No newline at end of file
+  sdkgen: [dotnet, go, java, nodejs, python, yaml, docs]
```

Comment:
docs is treated as a sdkgen backend, so I though this made sense. Let me know if you think this should be different. 

I figured the top level `docs` section is for changes to our docs (not docs generation).

Reply:
This makes sense ✅